### PR TITLE
Follow best practices naming in unit tests

### PR DESCRIPTION
**Pull request by Youssef1313**
Supports https://github.com/dotnet/samples/pull/1111

### CODE REVIEW

**Comment by Youssef1313** [comment]
I think more examples need to be updated to follow the naming best practice. so I'm adding [WIP] until I fix all of them.

File: docs/core/testing/unit-testing-with-nunit.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ title: Unit testing C# with NUnit and .NET Core
 description: Learn unit test concepts in C# and .NET Core through an interactive experience building a sample solution step-by-step using dotnet test and NUnit.
 author: rprouse
 ms.date: 08/31/2018
+dev_langs: 
+  - "csharp"
```
Comment (Thraka) [inline docs/core/testing/unit-testing-with-nunit.md]: Do you want to make sure all files have `dev_langs` field? I noticed that some more are missing the value.

**Review by Thraka — APPROVED** [review]
LGTM thanks! I did notice one thing that could be improved. Please see the note.

File: docs/core/testing/unit-testing-with-nunit.md (hunk 1 above)
Comment (pkulikov) [reply docs/core/testing/unit-testing-with-nunit.md]: If there's only one language in use, that field is not necessary, I guess.

Comment (Youssef1313) [reply docs/core/testing/unit-testing-with-nunit.md]: Yes, if it's only one language then it's not necessary.

Comment (mairaw) [reply docs/core/testing/unit-testing-with-nunit.md]: Correct, the field is not needed when there's only one language. We only added when we need to make the language switch appear. I'd actually remove it here.

Comment (Youssef1313) [reply docs/core/testing/unit-testing-with-nunit.md]: Okay, @mairaw. I'm going to remove it from the files where it exists :)

**Comment by Thraka** [comment]
Love it. Thanks!

**Pull request merged**